src/Illuminate/Foundation/Application.php

```diff
@@ -1051,7 +1051,95 @@ public function setLocale($locale)
      */
     public function isLocale($locale)
     {
-        return $this->getLocale() == $locale;
+        return $this->getLocale() === $locale;
+    }
+
+    /**
+     * Get default application locale
+     *
+     * @return string
+     */
+    public function getDefaultLocale()
+    {
+        return config('app.fallback_locale');
```

Comment:
Actually from my perspective having both locale and fallback_locale is a bit confusing. May be can remove fallback_locale at all and use app.locale for CLI also?